tests/test_web_api.py

```diff
@@ -23,36 +23,20 @@
     from urlparse import urlparse
 
 fake_pipeline_id = uuid4()
-fake_email_id = uuid4()
 
 
-def fake_getter(attrname):
-    return attrname
-
-
-def fake_add_redis_data_to_job_data(job_data):
-    return job_data
-
-
-def fake_delete_eregs_job(job_id):
-    return None
-
-
-def fake_redis_job(args, timeout=60*30, result_ttl=-1):
+def _fake_redis_job(cmd, args, timeout=60*30, result_ttl=-1, depends_on=None):
     return type("FakeRedisJob", (object, ), {"id": fake_pipeline_id})
```

Comment:
No biggie, but would
```python
return namedtuple("FakeRedisJob", ['id'])(fake_pipeline_id)
```
be easier to read? I don't think too many people know about `type`